source/solvers/postprocessors_smoothing.cc

```diff
@@ -98,7 +105,7 @@ const TrilinosWrappers::MPI::Vector &
 PostProcessorSmoothing<dim, VectorType>::solve_L2_projection()
 {
   // Solve the L2 projection system
-  const double linear_solver_tolerance = 1e-15;
+  const double linear_solver_tolerance = 1e-12;
```

Comment:
Why this is not the same as the parameter from prm?

Reply:
Because here the matrix is always easy to use. We should always solve it as well as possible, it's a trivial system to solve.